pytasa/fundamental.py

```diff
@@ -8,7 +8,407 @@
 
 import numpy as np
 
-# FIXME: this is a good place to put things like phase velocity 
-#        calculation or tensor rotation. Basic rule should be that
-#        the functions take a 6x6 np. array (or matrix?) as input
-#        and return something. We can then build an OO interface on top
+def _velo(X,rh,C):
+    """PHASE-VELOCITY SURFACES IN AN ANISOTROPIC MEDIUM
+       revised April 1991
+           X(3) - DIRECTION OF INTEREST
+           RHO - DENSITY
+           V - PHASE VELOCITIES 0,1,2= P,S,SS)
+           EIGVEC[3,3] - eigenvectors stored by columns
+       Translated to MATLAB by James Wookey   
+       Translated to Python by Alan Baird      
+    """
+    ijkl = np.array([[0,5,4],
+                     [5,1,3],
+                     [4,3,2]])
+    
+    gamma = np.array([[ X[0],  0.0,  0.0,  0.0, X[2], X[1]],
+                      [  0.0, X[1],  0.0, X[2],  0.0, X[0]],
+                      [  0.0,  0.0, X[2], X[1], X[0],  0.0]])
+    T = np.dot(np.dot(gamma,C),np.transpose(gamma))
+    
+    # determine the eigenvalues of symmetric tij
+    EIVAL,EIVEC = np.linalg.eig(T)
+    
+    # calculate velocities and sort
+    V_RAW = (np.sqrt(EIVAL/rh))*10.
+    IND=np.argsort(V_RAW) 
+    #np.argsort() assumes ascending sort, reverse indices for descending IND[::-1]
+    V=V_RAW[IND[::-1]]
+    EIGVEC = EIVEC
+    EIGVEC=EIVEC[:,IND[::-1]]
+
+    return V, EIGVEC
+    
+def _isIsotropic(C,tol):
```

Comment:
Minor issue: can we stick to `underscores_like_this` for function names, not `camelCaseLikeThis`?